src/tools/dart-run.md

```diff
@@ -129,13 +129,30 @@ $ dart run foo arg1 arg2
 ## Debugging
 
 To enable debugging, pass one or more debugging options.
-Here's an example of enabling [`assert` statements][assert]:
+
+To enable [`assert` statements][assert],
+specify the `--enable-asserts` flag:
 
 ```terminal
 $ dart run --enable-asserts tool/debug.dart
 ```
 
-See `dart run --help` for details.
+To enable debugging through [Dart DevTools](/tools/dart-devtools),
+a set of debugging and performance tools,
+specify the `--observe` flag.
+Optionally add `--pause-isolates-on-start`
+to break execution at the start of the specified script:
+
+```terminal
+$ dart run --pause-isolates-on-start --observe tool/debug.dart
+```
+
+Open Devtools
```

Comment:
The _"Open DevTools with the printed **Dart DevTools debugger and profiler** link."_-phrasing is a bit fuzzy.

Can we say something like "Open DevTools by copying the Dart DevTools debugger and profiler URL printed in the terminal, and pasting it into a Chrome browser window"> ?

Reply:
**issue:** For clarity, I would replace this with:
```
Copy the URL in the response and paste it into a Chrome browser window. Chrome then displays the Dart DevTools Debugger and Profiler.
```